src/infra/main/template.yaml

```diff
@@ -171,6 +186,16 @@ Resources:
     Properties:
       FunctionName: !Sub "${AWS::StackName}-InterventionsProcessorFunction"
       Description: "Processes intervention events and updates account's intervention status."
+      DeploymentPreference:
+        Type: !If
+            - IsProd
+            - !Ref LambdaDeploymentPreference
+            - AllAtOnce
+        Alarms:
+          - !Ref InterventionsProcessorFunctionErrorsAlarm
```

Comment:
Small thing/just a suggestion: I wonder if it's be easier to manage those alarms if we put canary in the name? since they are not used for anything else? cc @TaherKapasi